plugin/phpactor.vim

```diff
@@ -52,6 +55,11 @@ function! phpactor#Complete(findstart, base)
         endfor
     endif
 
+    if !empty(issues)
+        echoe join(issues, ', ')
+        return -3
```

Comment:
It might be somehow possible that the issue is non-blocking and we still have suggestions, should probably just continue.